src/legacy/core_plugins/vis_type_metric/public/metric_vis_type.test.ts

```diff
@@ -44,7 +40,7 @@ describe('metric_vis - createMetricVisTypeDefinition', () => {
   let vis: Vis;
 
   beforeAll(() => {
-    visualizationsSetup.createReactVisualization(createMetricVisTypeDefinition());
+    npSetup.plugins.visualizations.createReactVisualization(createMetricVisTypeDefinition());
```

Comment:
line 70 below is failing in this test too, because of the `@ts-ignore`, but updating it is going to require some additional mocking